charts/greenhouse/templates/plugin-alerts.yaml

```
@@ -21,6 +21,11 @@ spec:
           name: ingress-ca-cert
     - name: global.greenhouse.baseDomain
       value: {{ .Values.global.dnsDomain }}
+    - name: alerts.caCert
+      valueFrom:
+        secret:
+          key: ca.crt
+          name: tls-alerts-{{ .Chart.Name }}-root-ca
```

```suggestion
          name: tls-alerts-{{ .Release.Namespace  }}-root-ca
```

.Chart.name would be `alerts` vs `ccloud` which is needed here.

`tls-alerts-ccloud-root-ca`

---

In this case it is for the `greenhouse` Organization and Chart Name and Release Namespace are the same, since the chart is build around the greenhouse org which automatically receives the greenhouse namespace.
But this your suggestion is more future proof.